carbonmark/components/pages/Users/ProfileHeader/index.tsx

```diff
@@ -1,51 +1,51 @@
 import { Trans } from "@lingui/macro";
 import { ProfileLogo } from "components/pages/Users/ProfileLogo";
 import { Text } from "components/Text";
+import { User } from "lib/types/carbonmark";
 import { FC } from "react";
 import * as styles from "./styles";
 
 type Props = {
+  carbonmarkUser: User | null;
   userName: string;
-  description?: string;
-  isCarbonmarkUser: boolean;
-  profileImgUrl?: string;
-  handle?: string;
 };
 
 export const ProfileHeader: FC<Props> = (props) => {
+  const isCarbonmarkUser = !!props.carbonmarkUser;
+
   return (
     <div className={styles.profileHeader}>
       <ProfileLogo
-        isCarbonmarkUser={props.isCarbonmarkUser}
-        profileImgUrl={props.profileImgUrl}
+        isCarbonmarkUser={isCarbonmarkUser}
+        profileImgUrl={props.carbonmarkUser?.profileImgUrl}
       />
       <div className={styles.profileText}>
         <div className={styles.titles}>
-          <Text t="h4">{props.userName}</Text>
-          {props.handle && (
+          <Text t="h4">{props.carbonmarkUser?.username || props.userName}</Text>
```

Comment:
I'm confused because it looks like here props.userName can actually be the handle?
https://github.com/KlimaDAO/klimadao/pull/1099/files#diff-4ffaaf4f61284f7106b9ea09e882f4c38d70b88eab545268a412b9f50f923c95R25

So its possible that we could render `Handle @Handle` ?

Reply:
This should never happen. 
If `props.userName` falls back to the handle, then the carbonmark user data exists.
In this case the display name (username) is shown first.